air-script/tests/aux_trace/aux_trace.masm

```diff
@@ -45,6 +62,16 @@ proc.compute_integrity_constraint_divisor
     # => [divisor_1, divisor_0, ...]
 end # END PROC compute_integrity_constraint_divisor
 
+# Procedure to evaluate each integrity constraints.
+#
+# All the 3 main and 2 auxiliary constraints are computed.
+# The result of each evaluation is kept in the stack, with the top of the stack
```

Comment:
minor nit: should probably be "... kept on the stack, ...".

Reply:
thanks, fixed